File: pr1-java/networking/src/main/java/pr1Java/networking/dataTransfer/SeatsSellingDto.java
package pr1Java.networking.dataTransfer;

import java.io.Serializable;

public class SeatsSellingDto implements Serializable {
    private final GameDto gameDto;
    private final String clientName;
    private final Integer seatsCount;

    public SeatsSellingDto(GameDto gameDto, String clientName, Integer seatsCount) {
        this.gameDto = gameDto;
        this.clientName = clientName;
        this.seatsCount = seatsCount;
    }

    public GameDto getGameDto() {
        return gameDto;
    }

    public String getClientName() {
        return clientName;
    }

    public Integer getSeatsCount() {
        return seatsCount;
    }

    @Override
    public String toString() {
        return "SeatsCountDto[" + getGameDto() + " " + getClientName() + " " + getSeatsCount() + "]";
    }
}

File: pr1-cs/Networking/Reflection/DataTransfer/SeatsSellingDto.cs
using System;

namespace Networking.Reflection.DataTransfer
{
    [Serializable]
    public class SeatsSellingDto
    {
        public SeatsSellingDto(GameDto gameDto, string clientName, int seatsCount)
        {
            GameDto = gameDto;
            ClientName = clientName;
            SeatsCount = seatsCount;
        }

        public GameDto GameDto { get; }
        public string ClientName { get; }
        public int SeatsCount { get; }

        public override string ToString()
        {
            return "SeatsCountDto[" + GameDto + " " + ClientName + " " + SeatsCount + "]";
        }
    }
}